pkg/operator/ceph/file/dependent.go

```diff
@@ -55,6 +57,14 @@ func cephFilesystemDependents(clusterdCtx *clusterd.Context, clusterInfo *client
 	nsName := fmt.Sprintf("%s/%s", filesystem.Namespace, filesystem.Name)
 	baseErrMsg := fmt.Sprintf("failed to get dependents of CephFilesystem %q", nsName)
 
+	_, err := cephclient.GetFilesystem(clusterdCtx, clusterInfo, filesystem.Name)
+	if err != nil {
+		if strings.Contains(err.Error(), "not found") {
```

Comment:
Did you find an error code to use instead of the string?

Reply:
> @travisn Not found，is there error code for it？

The simplest way to see it might be to try printing the full err struct and see if it has any fields with an error code
```suggestion
		logger.Infof("GetFilesystem err: %+v")
		if strings.Contains(err.Error(), "not found") {
```

Reply:
ceph-file-controller: GetFilesystem err: exit status 2